library/aesce.c

```diff
@@ -202,6 +203,13 @@ static uint8x16_t aesce_decrypt_block(uint8x16_t block,
     block = veorq_u8(block, vld1q_u8(keys));
 
     return block;
+#else
+    (void) block;
+    (void) keys;
+    (void) rounds;
+
+    return MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED;
```

Comment:
I prefer the second one. If the feature is not enabled, we should return an error rather than `success` and generating a incorrect but meaningful result.